smoke-tester/src/main.rs

```diff
@@ -378,8 +397,14 @@ impl<'dut> TestTracker<'dut> {
             }
             println!();
 
-            let join_result =
-                std::thread::scope(|s| s.spawn(|| handle_dut(self, definition)).join());
+            let join_result = std::thread::scope(|s| {
+                s.spawn(|| {
```

Comment:
We still spawn a scoped thread with a local runtime to catch eventual panics gracefully.